Can't export functions that have 'PackedStringArray' or 'PackedByteArray' parameters

Hello, I'm trying to connect my node to the `request_completed` signal of an `HTTPRequest` node, the function in which I'm connecting the signal is like this:

```
@:export
function httpQueryCompleted(res: Int, response_code: Int, headers: PackedStringArray, body: PackedByteArray) { }
```

but when I try to compile it, it throws the following errors:

```
./src/example/HxExample.cpp(2244): error C2039: 'PackedByteArray_obj': is not a member of 'godot'
\bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
./src/example/HxExample.cpp(2244): error C3083: 'PackedByteArray_obj': the symbol to the left of a '::' must be a type
./src/example/HxExample.cpp(2244): error C2039: '___binding_callbacks': is not a member of 'godot'
\bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
./src/example/HxExample.cpp(2251): error C2039: 'PackedStringArray_obj': is not a member of 'godot'
bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
./src/example/HxExample.cpp(2251): error C3083: 'PackedStringArray_obj': the symbol to the left of a '::' must be a type
./src/example/HxExample.cpp(2251): error C2039: '___binding_callbacks': is not a member of 'godot'
\bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
./src/example/HxExample.cpp(2458): error C2039: 'PackedByteArray_obj': is not a member of 'godot'
\bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
./src/example/HxExample.cpp(2458): error C3083: 'PackedByteArray_obj': the symbol to the left of a '::' must be a type
./src/example/HxExample.cpp(2458): error C2039: '___binding_callbacks': is not a member of 'godot'
\bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
./src/example/HxExample.cpp(2465): error C2039: 'PackedStringArray_obj': is not a member of 'godot'
\bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
./src/example/HxExample.cpp(2465): error C3083: 'PackedStringArray_obj': the symbol to the left of a '::' must be a type
./src/example/HxExample.cpp(2465): error C2039: '___binding_callbacks': is not a member of 'godot'
\bin\include\godot/variant/__Variant.h(13): note: see declaration of 'godot'
```

--- a/src/godot/macros/ArgumentMacros.hx
+++ b/src/godot/macros/ArgumentMacros.hx
@@ -98,6 +98,39 @@ class ArgumentMacros {
                             );
                             v;
                         };
+
+                    case 'PackedByteArray':
+                        macro {
+                            var p = new PackedByteArray();
+
+                            (untyped __cpp__(
+                                'memcpy({4}->opaque, (uint8_t *)(*((({0} **){1})+{2})+{3}), {5})',
+                                $i{ptrSize},
+                                $i{_args},
+                                $v{_index},
+                                $v{_offset},
+                                p,
+                            PackedByteArray.PACKEDBYTEARRAY_SIZE
+                            ));
+                            p;
+                        }
+
+                    case 'PackedStringArray':
+                        macro {
+                            var p = new PackedStringArray();
+
+                            (untyped __cpp__(
+                                'memcpy({4}->opaque, (uint8_t *)(*((({0} **){1})+{2})+{3}), {5})',
+                                $i{ptrSize},
+                                $i{_args},
+                                $v{_index},
+                                $v{_offset},
+                                p,
+                                PackedStringArray.PACKEDSTRINGARRAY_SIZE
+                            ));
+                            p;
+                        }
+
                     default: {
                         var identBindings = '&::godot::${_d.name}_obj::___binding_callbacks';
                         macro {

